README.md

```diff
@@ -1,42 +1,83 @@
+| Directory | Description |
+|------------|-------------|
+| **`board/`** | Platform-specific configuration: patches and board support for SBCs and handhelds. |
+| **`buildroot/`** | The Buildroot framework used to assemble REG Linux images. |
+| **`configs/`** | Build configurations defining what packages and options are enabled per device. |
+| **`package/`** | The main system and emulator packages (Makefiles, patches, configuration). |
+| **`scripts/`** | Utilities and helper scripts for building, flashing, or managing devices. |
 
- - :wrench: If you want to help with development, [we accept PRs](https://makeapullrequest.com/)
- - :art: No need to be a developer, you can also [help with ES translations](https://wiki.batocera.org/help_with_translation), talk about our project on [Youtube](https://www.youtube.com/channel/UClFpqHKoXsOIV-GjyZqoZcw/featured) or [Twitter](https://twitter.com/REG_linux/), create [themes for EmulationStation](https://wiki.batocera.org/themes)
- - :dollar: If you like REG Linux, you will soon be able to help us with donations, it's always appreciated!
+A more detailed explanation of the build process and directory layout is available in the [Wiki → Developer Guide](https://wiki.reglinux.org/dev-guide/).
 
-## Directory navigation
+---
 
- - `board` Platform-specific build configuration. This is where to include special patches/configuration files needed to have particular components work on a particular platform. It is instead encouraged to apply patches at the location of the package itself, but this may not always be possible.
- - `buildroot` Buildroot, the tool used to create the final compiled images. For newcomers, you can safely ignore this folder. Compilation instructions can be found [on Batocera wiki](https://wiki.batocera.org/compile_batocera.linux).
- - `configs` Build flags, which define what components will be built with your image depending on your chose architecture. If you're trying to port REG Linux to a new architecture (device, platform, new bit mode, etc.) this is the file you'll need to edit. More information on [the build configuration section on the buildroot compiling page](https://wiki.batocera.org/batocera.linux_buildroot_modifications#define_your_configuration).
- - `package` The "meat and potatoes" of the operating system. This is where the majority of emulator data, config generators, core packages, system utilities, etc. all go into. This is the friendliest place to start dev-work for new devs, as most of it is handled by Python and Makefile.
- - `scripts` Various miscellanous scripts that handle aspects external to REG Linux, such as the report data sent to the [batocera compatibility page](https://batocera.org/compatibility.php) or info about the Bezel Project.
+### 🧑‍💻 Build Instructions
 
-A cheatsheet of notable files/folders can be found [on the wiki](https://wiki.batocera.org/notable_files).
+```bash
+git clone https://github.com/REG-Linux/REG-Linux.git
```

Comment:
Recursion is necessary (buildroot).
git clone --recursive https://github.com/REG-Linux/REG-Linux.git

Reply:
Good catch, fixed in latest commit